enterprise/cli/provisionerdaemons.go

```diff
@@ -291,6 +315,13 @@ func (r *RootCmd) provisionerDaemonStart() *clibase.Cmd {
 			Value:       clibase.StringArrayOf(&logFilter),
 			Default:     "",
 		},
+		{
+			Flag:        "prometheus-address",
+			Env:         "CODER_PROMETHEUS_ADDRESS",
+			Description: "The bind address to serve prometheus metrics.",
+			Value:       clibase.StringOf(&prometheusAddress),
+			Default:     "127.0.0.1:2112",
+		},
```

Comment:
non-blocking: I'm in two minds as to whether it should be `CODER_PROVISIONERD_PROMETHEUS_ADDRESS` for consistency with other provisionerd env vars, or just `CODER_PROMETHEUS_ADDRESS` for consistency with coderd.

Reply:
I had this concern too, but then noticed that we already "support" `CODER_PROMETHEUS_ADDRESS` everywhere, so extra +1 point for backward compatibility even though it didn't work before?

Reply:
I'm sold. :shipit: